src/test/java/net/openhft/chronicle/analytics/internal/MuteAnalyticsTest.java

```diff
@@ -5,7 +5,7 @@
 
 import java.util.Collections;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
mockwebserver still brings junit 4 dependencies in, but we should always use Junit5 ones

Reply:
can we exclude?